packages/config/config/devices/0x027a/zen15.json

```diff
@@ -79,53 +93,97 @@
 		},
 		"151": {
 			"label": "Power Report Value Threshold",
-			"description": "When to report power consumption to your controller",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 65535,
 			"defaultValue": 50,
-			"unsigned": true
+			"unsigned": true,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				}
+			]
 		},
 		"152": {
 			"label": "Power Report Percentage Threshold",
-			"description": "How to report power consumption to your controller by percentage rate",
-			"valueSize": 1,
+			"valueSize": 2,
 			"minValue": 0,
```

Comment:
`"unit": "%"`?

Reply:
@blhoward2?

Reply:
So the manual is very confusing on this one. The description suggests it is 1-100 and a percentage, but the manual is also clear that the allowed range is 0-255. I'm fine with however you want to handle it. Do you just want to make it 0-100 and a %?

Parameter 152: Choose how you want your Power Switch
to report power use to your hub and associated devices by
percentage rate. The number entered as value
corresponds to the percentage in power usage change the
appliance needs to go over for the event to be reported.
So if 10% is entered by default, the Power Switch will
report any change in power use over 10% (whether it’s at
least 10% more or 10% less power use compared to
previous report).
Values: 0 – 255; 0 – disabled (it will not report power
consumption based on percentage change); 10 – default
setting. Size: 1 byte dec.

Reply:
I don't think that in this case 0-255 means that it is not a percentage. I understand 255 as "if the value changes by more than 255% of the last value, send a report".